ROADMAP.md

```diff
@@ -22,20 +20,14 @@ General bugfixing and clean-up tasks are assumed.  This lists major features nee
 ## 0.1
 * Associate resources with resources
 * Associate types via nesting in a block
-* Parse networking rules
-* Parse filesystem and ephemeral object rules
```

Comment:
We handle filesystem rules, unless you mean something else.

Reply:
I'm removing it from the list of future work, since it's done.